sources/packages/web/src/assets/css/formio-shared.scss

```diff
@@ -38,7 +38,6 @@
   font-size: 12px;
   line-height: 18px;
   letter-spacing: 0.5px;
-  text-transform: uppercase;
```

Comment:
As talked with @hellolynn-tbtb we no longer need `uppercase` transformation because it causes issues with the accessibility.